pkg/preparer/orchestrate.go

```diff
@@ -418,7 +420,9 @@ func (p *Preparer) installAndLaunchPod(pair ManifestPair, pod Pod, logger loggin
 			}
 		} else {
 			// TODO: do this in a transaction
-			err = p.podStore.WriteRealityIndex(pair.PodUniqueKey, p.node)
+			// TODO dai - what do I do with the errors?
```

Comment:
hmm this is a tricky one. At this stage in the code, we've already started the pod and we're trying to write to reality so that we don't do it a second time, which could be bad. It's pretty important that the write succeeds.

Probably this whole bit of code from `transaction.New()` to `transaction.Commit()` should be retried until success with an exponential backoff `time.Sleep` in between failures. That way we guarantee this goroutine won't take any additional action until the write is successful